Services/Dashboard/Administration/classes/class.ilDashboardSortationTableGUI.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+class ilDashboardSortationTableGUI extends ilTable2GUI
+{
+    private ilAccessHandler $access;
+    private \ILIAS\UI\Renderer $uiRenderer;
+    private \ILIAS\UI\Factory $uiFactory;
+    private ilPDSelectedItemsBlockViewSettings $viewSettings;
+    private ilDashboardSidePanelSettingsRepository $side_panel_settings;
+
+    public function __construct($a_parent_obj, $a_parent_cmd)
+    {
+        global $DIC;
+        $this->uiFactory = $DIC->ui()->factory();
+        $this->uiRenderer = $DIC->ui()->renderer();
+        $this->viewSettings = new ilPDSelectedItemsBlockViewSettings($DIC->user());
+        $this->side_panel_settings = new ilDashboardSidePanelSettingsRepository();
+        parent::__construct($a_parent_obj, $a_parent_cmd);
+        $this->lng->loadLanguageModule('mme');
+        $this->initColumns();
+        $this->setFormAction($this->ctrl->getFormAction($this->parent_obj));
+        $this->addCommandButton('saveSettings', $this->lng->txt('save'));
+        $this->setRowTemplate(
+            "tpl.dashboard_sortation_row.html",
```

Comment:
The quotation through the whole PR seems inconsistent.
Please use singel quotes `'` if able.